PetApp/PetApp/controllers/EventInfoViewController.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  EventInfoViewController.swift
+//  PetApp
+//
+//  Created by Пользователь on 07.03.2021.
+//
+
+import Foundation
+import UIKit
+
+class EventInfoViewController: UIViewController {
+    
+    var event: Event
+    
+    init(event: Event) {
+        self.event = event
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureView()
+    }
+    
+    
+    private func configureView() {
+        let screenHeight = UIScreen.main.bounds.height
+        let screenWidth = UIScreen.main.bounds.width
+        
+        if((event.image?.isEmpty) != nil) {
```

Comment:
Нужно избегать дублирования проверок. Здесь guard ниже дублирует это условие, хотя достать имедж из опционала можно прямо здесь:
```suggestion
        if let image = event.image, !image.isEmpty {
```